packages/client/src/app/pages/forms/edit/builder/tabs/layout-editor/field-list/field-group/field-group.styles.ts

```diff
@@ -1,6 +1,10 @@
 import { spacings } from '@ff-client/styles/variables';
 import styled from 'styled-components';
 
+export const FieldGroupWrapper = styled.div``;
+
+export const GroupTitle = styled.h2``;
+
 export const List = styled.div`
```

Comment:
Again, not nit picking but why pull in all components when we only use `div` and `h2`. 

`import { div, h2} from 'styled-components';`